sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -108,3 +108,15 @@
 --  #IfVitalsDatesNeeded
 --    desc: Change date from zeroes to date of vitals form creation.
 --    arguments: none
+
+#IfMissingColumn onetime_auth scope
+ALTER TABLE `onetime_auth` ADD `scope` tinytext COMMENT 'context scope for this token';
+#EndIf
+
+#IfMissingColumn onetime_auth profile
+ALTER TABLE `onetime_auth` ADD `profile` tinytext COMMENT 'profile of scope for this token';
+#EndIf
+
+#IfMissingColumn onetime_auth onetime_actions
+ALTER TABLE `onetime_auth` ADD `onetime_actions` text COMMENT 'JSON array of actions that can be performed with this token';
+#EndIf
```

Comment:
If this ends up going back to rel-703, then would migrate this to 7_0_2-to-7_0_3_upgrade.sql in master and rel-703 and increment $v_database in rel-703 to 506

Reply:
was doing when I create backport, this is how I have to schedule my brain tasks, thanks!